solver/src/main/java/org/chocosolver/solver/variables/Task.java

```diff
@@ -29,7 +28,7 @@
  * @author Jean-Guillaume Fages
  * @since 04/02/2013
  */
-public class Task {
+public class Task implements ICause {
```

Comment:
I'm confused: the Task object was initially designed to maintain the relationship between start, duration and end during propagation. Now, if the relation is maintained explicitly by propagators, what is the purpose of Task ?